Engine/Assets/Skeleton.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Engine.AssetsData;
+using LinearAlgebra;
+using SharpDX.Direct3D11;
+using System.Collections.Generic;
+using System;
+using SharpDX;
+using Assimp;
+
+namespace Engine
+{
+    public class Skeleton : BaseAsset
+    {
+        public Matrix4x4f InverseRootTransform = Matrix4x4f.Identity;
+        public List<Bone> Bones = new List<Bone>();
+    }
+
+    public class Bone
```

Comment:
add todo to split this class into business class and data class instead of using the same one both in Skeleton and SkeletonData